scripts/install/CreateItemCommentIndex.php

```
@@ -0,0 +1,44 @@
+namespace oat\taoAdvancedSearch\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\oatbox\reporting\Report;
+use oat\taoAdvancedSearch\model\Comment\ItemCommentIndexManager;
+
+/**
+ * Ensures the item-comments Elasticsearch index exists (DI wires the adapter).
+ */
+class CreateItemCommentIndex extends InstallAction
+{
+    public function __invoke($params = [])
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$params'. (undefined)

(UnusedFormalParameter)